### PR TITLE
Enable passing of DragEvent to gridstack dropped event

### PR DESCRIPTION
**Pull request by onepartsam**
### Description
Enhancement done to allow the passing of DragEvent to gridstack dropped event, allowing access to dataTransfer property. A demo in Angular is included to showcase this enhancement.

This enhancement is related to issue #1817

### CODE REVIEW

File: demo/drag-drop-dataTransfer.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/**
```
Comment (adumesny) [inline demo/drag-drop-dataTransfer.ts:1]: thanks for putting an example, but user can't simply run this in a browser to test/play with it. do you mind putting a generic JS examples like the others ?
(I happen to use Angular myself but most don't as this lib isn't tied to any framework). thanks!

Comment (onepartsam) [reply demo/drag-drop-dataTransfer.ts:1]: Hi, I have just included a generic JS example.

**Comment by adumesny** [comment]
thank you!

**Comment by onepartsam** [comment]
Hi @adumesny, thank you. May I know when you are going to make a new release with this modification?

**Comment by adumesny** [comment]
I like to have a few fixes in per release if possible... I have another one pending, so maybe in a couple weeks ?

**Pull request merged**